scripts/lean-pr-testing-comments.sh

```diff
@@ -0,0 +1,106 @@
+## Create comments and labels on a Lean 4 PR after CI has finished on a `lean-pr-testing-NNNN` branch.
+set -e
+
+# This is not meant to be run from the command line, only from CI.
```

Comment:
```suggestion
# TODO: The whole script ought to be rewritten in javascript, to avoid having to use curl for API calls.
#
# This is not meant to be run from the command line, only from CI.
```